apps/chat/src/components/Common/AgentAndToolsetSelector/AgentAndToolsetModal.tsx

```diff
@@ -374,6 +374,7 @@ const AgentAndToolsetModalView = ({
             <TalkToNotFound
               isMyWorkspace={isMyWorkspace}
               onOpenMarketplaceTab={handleSetScopeTab}
+              isSearchMode={!searchTerm}
```

Comment:
search mode when search term is not empty
```suggestion
              isSearchMode={!!searchTerm}
```